packages/kit/src/components/Hardware/ConnectionTroubleShootingAccordion.tsx

```
@@ -0,0 +1,258 @@
+              </>
+            )}
+          </Accordion.Trigger>
+          <Accordion.HeightAnimator animation="quick">
+            <Accordion.Content
+              unstyled
+              animation="quick"
+              enterStyle={{ opacity: 0 }}
+              exitStyle={{ opacity: 0 }}
+            >
+              <Stack
+                role="list"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<ol>
<ul>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)